api/infrastructure/v1alpha2/linodecluster_conversion.go

```diff
@@ -0,0 +1,3 @@
+package v1alpha2
+
```

Comment:
Can we add a comment here like
`// Hub marks LinodeCluster as a conversion hub.`?

Reply:
Updated